### PR TITLE
added images and input

### PR DESCRIPTION
**Pull request by srijani07**
Images:
image
image

Input:
image

### CODE REVIEW

File: Images/images.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.responsive-image {
+  max-inline-size: 20%;
+  block-size: auto;
+  object-fit: contain;
```
Comment (mohammadkhizerkhan) [inline Images/images.css:4]: use object-fit:cover
image
image

**Review by ayush65 — APPROVED** [review]

**Comment by JowelTisso** [comment]
@srijani07 Why is your 3 merged PR not reflecting in dev branch. I was not able to see any code there.

File: Images/images.css (hunk 1 above)
Comment (JowelTisso) [inline Images/images.css:4]: Suggestion : max-inline-size property is mostly useful when using writing mode I guess. But its ok if it works for you.

**Review by JowelTisso — APPROVED** [review]

File: Images/images.css (hunk 1 above)
Comment (srijani07) [reply Images/images.css]: rectified, thanks!

Comment (srijani07) [reply Images/images.css]: hey, yes! max-inline-size works with writing mode.

**Pull request merged**